user/views.py

```diff
@@ -69,4 +69,94 @@ def post(self, request):
 
         except KeyError:
             return JsonResponse({'message':'KEYERROR'}, status=400)
-                
\ No newline at end of file
+
+class UserInfoView(View):
+    @Login_required
+    def get(self, request):
+        user = User.objects.get(id=request.user.id)
+
+        return JsonResponse({'name':user.name})
+
+class LikeProductView(View):
+    @Login_required
+    def post(self, request):
+        user_data = json.loads(request.body)
+
+        if not LikeProduct.objects.filter(product_id = user_data['product_id'], user_id = request.user.id).exists():
+            LikeProduct.objects.create(
+                user    = request.user,
+                product = Product.objects.get(id = user_data['product_id'])
+            ).save
+
+            return JsonResponse({'message':'ADD_LIKE_PRODUCT'}, status=200)
+
+        else:
+            delete_product = LikeProduct.objects.get(product_id = user_data['product_id'], user_id = request.user.id)
+            delete_product.delete()
+
+            return JsonResponse({'message':'DELETE_LIKE_PRODUCT'}, status=200)
+
+    @Login_required 
+    def get(self, request):
+        like_list    = []
+        like_product = LikeProduct.objects.filter(user_id = request.user.id).prefetch_related("product")
+        
+
+        for product in like_product:
+            data = {
+                "id"           : product.product.id,
+                "product_line" : product.product.product_line,
+                "name"         : product.product.name,
+                "price"        : product.product.price,
+                "sale_price"   : product.product.sale_price,
+                "hash_tag"     : product.product.hash_tag,
+                "images"       : product.product.image_set.first().image_url,
+                "sale"         : product.product.productflag_set.first().sale_flag,
+                "gift"         : product.product.productflag_set.first().gift_flag
+            }
+
+            like_list.append(data)
+
+        return JsonResponse({'like_list' : like_list})
+
+class CartProductView(View):
+    @Login_required
+    def post(self, request):
+        user_data = json.loads(request.body)
+        try:
+            if not CartProduct.objects.filter(product_id = user_data['product_id'], user_id = request.user.id).exists():
+                CartProduct.objects.create(
+                    user    = request.user,
+                    product = Product.objects.get(id = user_data['product_id'])
+                ).save
+
+                return JsonResponse({'message':'ADD_CART_PRODUCT'}, status=200) 
+            else:
+                return JsonResponse({'message': 'ALREADY_EXIST_PRODUCT'}, status=400)
```

Comment:
마찬가지로 `else ` 없이 return 구문은 한단락 앞당겨주세요
```suggestion
            
            return JsonResponse({'message': 'ALREADY_EXIST_PRODUCT'}, status=400)
```